### DISCUSSION TITLE
how can I make static heatmap

### DISCUSSION THREAD

**Discussion by didi9417** [openlayers/openlayers#15231] [Q&A]
I'm use openlayers 7.3.0 on Javascript and This my source

![image](https://github.com/openlayers/openlayers/assets/89561822/35022d1e-077e-49e5-85e9-981b894973af)

It is work well but I want my HeatMap Layer not affect zoom event like [this example](https://developers.arcgis.com/javascript/latest/sample-code/visualization-heatmap-static/) 

how can I make static heatmap?

**Comment by jahow** [comment]
A solution might be to make the `radius` and `blur` parameters change according to the resolution so that they stay constant in real world units, but I'm not sure it'll work exactly like in the provided example.
